### PR TITLE
Translate "Ruby 2.6.3 Released" (ko)

### PR DESCRIPTION
**Pull request by Jiwoong**
Ref #2023 

The actual diff is: aa0d008

Thank you

### CODE REVIEW

**Comment by riseshia** [comment]
Could you rebase master for ci check? 🤔

**Comment by Jiwoong** [comment]
It seems that `.travis.yml` [uses ruby 2.6.2](https://github.com/ruby/www.ruby-lang.org/blob/6b26f6cc8b30c5f1dfb8a666d56358345fd5d57a/.travis.yml#L3) even though `Gemfile` [uses ruby 2.6.3](https://github.com/ruby/www.ruby-lang.org/blob/6b26f6cc8b30c5f1dfb8a666d56358345fd5d57a/Gemfile#L2)

Should I open another PR to fix failed CI or just commit to bump travis rvm ruby version?

**Comment by yous** [comment]
@Jiwoong Yes, feel free to open a new PR.

File: ko/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+title: "루비 2.6.3 릴리스"
+author: "naruse"
+translator: "jiwoong"
+date: 2019-04-17 00:00:00 +0000
+lang: ko
+---
+
+루비 2.6.3이 릴리스 되었습니다.
+
+이 릴리스는 일본의 새로운 연호 "令和" (레이와)에 대한 지원을 추가합니다.
```
Comment (yous) [inline ko/news/_posts/2019-04-17-ruby-2-6-3-released.md]:
```suggestion
이 릴리스는 일본의 새로운 연호 '令和'(레이와)에 대한 지원을 추가합니다.
```

**Pull request merged**